cg/apps/demultiplex/sample_sheet/create.py

```diff
@@ -27,7 +27,7 @@ def create_sample_sheet(
 
     if run_parameters.flow_cell_type != FlowCellType.NOVASEQ:
         message = (
-            f"Can only demultiplex novaseq with cg. Found type {run_parameters.flow_cell_type}"
+            f"Can only demultiplex Novaseq with Cg. Found type {run_parameters.flow_cell_type}"
```

Comment:
I think the abbreviation Cg might be a bit too cryptic as to what it refers to. Maybe we can expand this string to be a bit more clear?

Reply:
Fixed